object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/framework/FrameworkSupport.java

```diff
@@ -0,0 +1,31 @@
+package org.checkerframework.checker.objectconstruction.framework;
+
+import com.sun.source.tree.NewClassTree;
+import org.checkerframework.framework.type.AnnotatedTypeMirror;
+
+public interface FrameworkSupport {
+
+  /**
+   * For a toBuilder routine, we know that the returned Builder effectively has had all the required
```

Comment:
Clarify this is called for any method _named_ `toBuilder`.  It's up to the `FrameworkSupport` object to figure out if it's the right one